dev_guide/projects.adoc

```diff
@@ -165,3 +165,29 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
+
+You can use node selectors in conjunction with labeled nodes to control pod placement.
+
+[NOTE]
+====
+OpenShift administrators can assign labels link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+
+OpenShift cluster administrators can set the cluster-wide default node selector to restrict pod placement to specific nodes. As an OpenShift developer, you can set a node selector on a project configuration to restrict nodes even further. This will be added to the cluster-wide node selector, meaning it will be more restrictive.
+
+To add a node selector, edit a pod configuration, and add the `nodeSelector` value:
+
+====
+----
+...
+nodeSelector:
+  type:user-node,region:east
```

Comment:
Proper YAML would look like:

```
nodeSelector:
  type: user-node
  region: east
```

I'm not sure I'd use this example though as the labels sound more cluster-admin-y. Maybe:

```
nodeSelector:
  disktype: ssd
```

I would prefer to link to kube docs (e.g. http://kubernetes.io/v1.1/docs/user-guide/node-selection/README.html) for a better in-context example. And would like to mention here that your labels are added to any the cluster-admin has specified by default / for your project.